validator/src/machine/consensus/epochProposed.ts

```diff
@@ -2,25 +2,34 @@ import type { ShieldnetProtocol } from "../../consensus/protocol/types.js";
 import type { VerificationEngine } from "../../consensus/verify/engine.js";
 import type { EpochRolloverPacket } from "../../consensus/verify/rollover/schemas.js";
 import type { EpochProposedEvent } from "../transitions/types.js";
-import type { MachineConfig, MachineStates, StateDiff } from "../types.js";
+import type { MachineStates, StateDiff } from "../types.js";
 
 export const handleEpochProposed = async (
```

Comment:
I would remove this method completely and the message is already verified (https://github.com/safe-research/shieldnet/pull/68/files#diff-7c2e07da90f8ad73f0cfdac7076dceb43f8da263edcd1b9bcd4e6b2db0eda712R110)